sass/_themes.scss

```diff
@@ -810,22 +814,26 @@ $themes: (
 
 :export {
     red: $color-red;
+    chart_rise_up: $COLOR_CHART_FALL_DOWN;
+    chart_fall_down: $COLOR_CHART_RISE_UP;
```

Comment:
besides this it would be better to have a more meaningful name, something like `light_candle_up`, `light_candle_down` and if it's the same color for light and dark, we can ignore the theme prefix